unity/README.md

```diff
@@ -4,6 +4,48 @@ This is Unity's fork of the .NET Runtime repository.
 
 The difference between this fork and the upstream repository should be as small as possible, with all of the differences specific to Unity. Our goal is to upstream as many changes made here as possible.
 
+## Building, testing, and debugging locally
+
+The CoreCLR runtime and class libraries can be built and tested locally for Windows, macOS, and Linux.
+
+### Building the CoreCLR runtime and class libraries
+
+To build locally, use the platform-specific script in .yamato/scripts:
+
+```
+> .yamato\scripts\build_windows.cmd x64 Debug
+```
+
+This will build the CoreCLR runtime and the class libraries for Windows, x64 architecture in the debug configuration.
+
+### Testing locally
+
+To test locally, use the platform-specific script in .yamato/scripts:
+
+```
+.yamato\scripts\test_windows.cmd x64 Debug
+```
+
+This will use the built artifacts from the build command above (Windows OS, x64 architecture, debug configuration) to run both Unity-specific tests and some tests we care about from CoreCLR.
+
+### Building and testing together
+
+Use the scripts in the unity/test-scripts directory to run the build and tests together:
+
+```
+unity/test-scripts/build_test_windows.cmd x64 Debug
+```
+
+### Using a debug build of the CoreCLR runtime in Unity
+
+Once the CoreCLR runtime has been built locally you can find the runtime artfacts the Unity player needs in `artifacts\bin\microsoft.netcore.app.runtime.win-x64\Debug\runtimes\win-x64`. Replace `Debug` with `Release` in this path for the Release configuration. It is important to copy all of the runtime and class library files together, as they need to stay in sync in order for the CoreCLR runtime to work properly. On Windows, the recursive directory copy (overwriting files in the destination) looks like this:
+
+```
+> xcopy /e /y artifacts\bin\microsoft.netcore.app.runtime.win-x64\Debug\runtimes\win-x64 <Unity player build directory>\CoreCLR
+```
+
+**Caveat:** It is _not_ possible to enable mixed mode debugging in Visual Studio and also debug the native code in coreclr.dll. Visual Studio blocks this workflow to prevent hangs. To debug in coreclr.dll in a Unity player, Native debugging must be selected in Visual Studio.
```

Comment:
Maybe suggest using WinDBG if mixed debugging in coreclr.dll is desired?

Reply:
Yeah, that is next on my list to try. I got it working with native-only debugging yesterday, but ran out of time to try mixed mode. Once I have details about that I'll document it.